qiskit/utils/measurement_error_mitigation.py

```diff
@@ -113,8 +113,13 @@ def build_measurement_error_mitigation_circuits(
         backend (BaseBackend): backend instance
         backend_config (dict, optional): configuration for backend
         compile_config (dict, optional): configuration for compilation
+        mit_pattern (List[List[int]]): Qubits on which to perform the
```

Comment:
It can be optional. so `(List[List[int]], optional)`.

Reply:
Changed to Optional